client/components/seo-preview-pane/index.jsx

```diff
@@ -47,7 +51,11 @@ const getPostImage = ( post ) => {
 	}
 
 	const imgElements = parseHtml( content ).querySelectorAll( 'img' );
-	const largeImage = get( find( imgElements, ( { width } ) => width >= PREVIEW_IMAGE_WIDTH ), 'src', null );
+	const largeImage = get(
```

Comment:
this looks more like a URL, and I think if we named it like that it would help make sense of the return statement…

```js
const imageUrl = get( … );

return imageUrl
	? `${ imageUrl }?s=${ PREVIEW_IMAGE_WIDTH }`
	: null;
```